src/model/OnmsMonitoredService.ts

```diff
@@ -45,6 +47,7 @@ export class OnmsMonitoredService implements IHasUrlValue {
     const service = new OnmsMonitoredService();
 
     service.id = Util.toNumber(data.id);
+    service.down = data.down;
```

Comment:
does this need to be coerced? or will it come through as an actual boolean?

Reply:
It does come through as an actual boolean. The Java returns a `boolean` and raw Json response is:

```json
{
  "service": [
    {
      "down":true,
      "notify":null,
      "status":"A",
      "lastGood":1651862554301,
      // ...
      "id":4
    }
  ]
}
```